src/server/appserver.cpp

```diff
@@ -350,6 +351,25 @@ AppServer::AppServer(int &argc, char **argv)
         }
     }
 
+    appStateValue = "";
```

Comment:
```suggestion
```

Reply:
`appStateValue` need to be initialized with a type (here a std::string) for `selectAppState` to know the type it should convert the db value to.